### PR TITLE
feature: added unit tests for sequelize movement endpoints

### PR DESCRIPTION
**Pull request by IOVgomezdn**
added unit tests for movement CRUD

https://rsklabs.atlassian.net/browse/TQAS-44

### CODE REVIEW

File: test/api-endpoints/sequelize/movement.route.sequelize.spec.ts [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  describe("Find", () => {
+    it("should find an existing movement", (done) => {
+      chaiAppServer.get(getURL(newMovementID)).end((err, res) => {
+        expect(res.body.id).to.equal(newMovementID);
+        done(err);
+      });
+    });
+
+    it("shouldn't find a movement whose id doesn't exist", (done) => {
+      chaiAppServer.get(getURL("130")).end((err, res) => {
```
Comment (gaston-areco-iov) [inline test/api-endpoints/sequelize/movement.route.sequelize.spec.ts]: Add random and larger search parameter

File: test/api-endpoints/sequelize/movement.route.sequelize.spec.ts [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+        .send({ amount: 200 })
+        .end((err, res) => {
+          expect(res.status).to.equal(204);
+          done(err);
+        });
+    });
+
+    it("shouldn't update a movement whose id doesn't exist", (done) => {
+      chaiAppServer
+        .patch(patchURL("187"))
```
Comment (gaston-areco-iov) [inline test/api-endpoints/sequelize/movement.route.sequelize.spec.ts]: Add random and larger search parameter

File: test/api-endpoints/sequelize/movement.route.sequelize.spec.ts [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    describe("Delete", () => {
+      it("should delete an existing movement", (done) => {
+        chaiAppServer.delete(deleteURL(newMovementID)).end((err, res) => {
+          expect(res.status).to.equal(204);
+          done(err);
+        });
+      });
+
+      it("shouldn't delete a non existing movement", (done) => {
+        chaiAppServer.delete(deleteURL("1734")).end((err, res) => {
```
Comment (gaston-areco-iov) [inline test/api-endpoints/sequelize/movement.route.sequelize.spec.ts]: Add random and larger search parameter

**Review by gaston-areco-iov — CHANGES_REQUESTED** [review]

File: test/api-endpoints/sequelize/movement.route.sequelize.spec.ts (hunk 2 above)
Comment (IOVgomezdn) [reply test/api-endpoints/sequelize/movement.route.sequelize.spec.ts]: done

File: test/api-endpoints/sequelize/movement.route.sequelize.spec.ts (hunk 1 above)
Comment (IOVgomezdn) [reply test/api-endpoints/sequelize/movement.route.sequelize.spec.ts]: done

File: test/api-endpoints/sequelize/movement.route.sequelize.spec.ts (hunk 3 above)
Comment (IOVgomezdn) [reply test/api-endpoints/sequelize/movement.route.sequelize.spec.ts]: done

**Review by gaston-areco-iov — APPROVED** [review]

**Pull request merged**